Lib/test/test_capi/test_getargs.py

```diff
@@ -531,29 +531,32 @@ def __bool__(self):
         raise NotImplementedError
 
 class Boolean_TestCase(unittest.TestCase):
-    def test_p(self):
-        from _testcapi import getargs_p
-        self.assertEqual(0, getargs_p(False))
-        self.assertEqual(0, getargs_p(None))
-        self.assertEqual(0, getargs_p(0))
-        self.assertEqual(0, getargs_p(0.0))
-        self.assertEqual(0, getargs_p(0j))
-        self.assertEqual(0, getargs_p(''))
-        self.assertEqual(0, getargs_p(()))
-        self.assertEqual(0, getargs_p([]))
-        self.assertEqual(0, getargs_p({}))
-
-        self.assertEqual(1, getargs_p(True))
-        self.assertEqual(1, getargs_p(1))
-        self.assertEqual(1, getargs_p(1.0))
-        self.assertEqual(1, getargs_p(1j))
-        self.assertEqual(1, getargs_p('x'))
-        self.assertEqual(1, getargs_p((1,)))
-        self.assertEqual(1, getargs_p([1]))
-        self.assertEqual(1, getargs_p({1:2}))
-        self.assertEqual(1, getargs_p(unittest.TestCase))
-
-        self.assertRaises(NotImplementedError, getargs_p, Paradox())
+    def test_p_and_P(self):
+        import _testcapi
+
+        for parsefn in (_testcapi.getargs_p, _testcapi.getargs_P):
+            with self.subTest(name=parsefn.__name__):
```

Comment:
```suggestion
            with self.subTest(parsefn.__name__):
```

If you want to only show the name, the first parameter of `self.subTest` is kind of a "title" so you can do it like this.